src/LazyArrays.jl

```diff
@@ -38,7 +38,7 @@ import FillArrays: AbstractFill
 
 import StaticArrays: StaticArrayStyle
 
-export Mul, MulArray, MulVector, MulMatrix,
+export Mul, MulArray, MulVector, MulMatrix, Add,
```

Comment:
Let's not export `Add` in case we decide to change its definition.

Reply:
Done in 75836a8eeb9ec245131e87736f669bdceb1c5e25